docs/utilities.md

```diff
@@ -104,6 +104,16 @@ import { isValidLandlinePhone } from '@brazilian-utils/brazilian-utils';
 isValidLandlinePhone('1130000000'); // true
 ```
 
+## isValidLicensePlate
```

Comment:
Maybe you forgot to add `pt-BR` docs.

Reply:
Ill be doing this right now, didnt see that.